pkgs/by-name/on/onionbalance/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  python313Packages,
+  fetchFromGitLab,
+}:
+
+python313Packages.buildPythonApplication rec {
+  pname = "onionbalance";
+  version = "0.2.4";
+
+  pyproject = true;
+  src = fetchFromGitLab {
+    domain = "gitlab.torproject.org";
+    owner = "tpo";
+    repo = "onion-services/onionbalance";
+    rev = version;
+    hash = "sha256-amwKP9LJ7aHPECNUNTluFpgIFSRLxR7eHQxBxW5574I=";
+  };
+  propagatedBuildInputs = with python313Packages; [
+    cryptography
+    pycryptodomex
+    pyyaml
+    setproctitle
+    stem
+    setuptools
```

Comment:
```suggestion
```
Per https://github.com/torproject/onionbalance/blob/main/requirements.txt it is not required